components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,291 +10,788 @@ toc = true
+### Listing Rules for a Project
 
 ```bash
-curl -s -H "api-token: $TOKEN" -d @policy.json -X PUT https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}?pretty
+curl -sH "api-token: $TOKEN" \
+  https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/projects/{project-id}/rules?pretty
 ```
 
-You can also make the change specifying the data inline:
+The output lists all rules for a specified project.
+
+### Getting a Project Rule
 
 ```bash
-curl -s -H "api-token: $TOKEN" -d '{"name":"Test Policy", "id":"test-policy", "members":["team:local:alpha","team:ldap:beta"], "statements": [{"actions": [ "iam:users:list"], "effect": "ALLOW"}]}' -X PUT https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}?pretty
+curl -sH "api-token: $TOKEN" \
+  https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/projects/{project-id}/rules/{rule-id}?pretty
 ```
 
-### Policy Membership
+This shows the details of a single project, selected by its ID.
 
-Chef Automate supports two methods for updating policy membership.
+### Creating a Project Rule
 
-Using PUT lets you **set** membership for the entire policy; see [Setting Policy Membership]({{< relref "iam-v2-api-reference.md#setting-policy-membership" >}}).
+Create a rule by composing JSON with all the necessary rule properties (see [Example Project Rule]({{< relref "iam-v2-api-reference.md#example-project-rule" >}})).
 
-Using POST lets you add members to (or remove members from) an existing membership list.
-For more information, see [Adding Members to Policies]({{< relref "iam-v2-api-reference.md#adding-members-to-policies" >}}) and [Removing Members from Policies]({{< relref "iam-v2-api-reference.md#removing-members-from-policies" >}}).
+Assuming you store it in the file "project-rule.json", pass it to `curl` to create the project rule:
 
-#### Setting Policy Membership
+```bash
+curl -sSH "api-token: $TOKEN" -d @project-rule.json -X POST \
+https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/projects/{project-id}/rules?pretty
+```
 
-From the command line, specifying a complete list of members is often simpler than keeping track of a policy's membership and adding or deleting specific members (whether users, team, or tokens).
-This command *replaces* the policy's membership.
+### Updating a Project Rule
 
 ```bash
-curl -s -H "api-token: $TOKEN" -X PUT -d '{"members":["team:local:admins","token:admin-token"]}' https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}/members?pretty
+curl -sSH "api-token: $TOKEN" -d @project-rule.json -X PUT \
+https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/projects/{project-id}/rules/{rule-id}?pretty
 ```
 
-#### Adding Members to Policies
+- Supply all of a rule's properties, not just the ones you wish to update.
+  Properties that you do not include are reset to empty values.
+- The rule ID and Type are immutable; both properties can only be set at creation time.
```

Comment:
Worth noting that due to this, if you'd like to change the rule, you'll need to delete it?